exercises/practice/leap/.docs/instructions.append.md

```diff
@@ -0,0 +1,7 @@
+# Hints
```

Comment:
We actually have a special `.docs/hints.md` file for this. See h-file-docs-hints-md

It requires some special formatting (an `## General` heading). See https://github.com/exercism/elixir/blob/main/exercises/practice/darts/.docs/hints.md for an example.

Reply:
I botched the doc link. I updated